OpenWhere/.github
# Replace old templates with new checklist templates

# Pull Request Description
This pull request replaces the original org-wide issue templates with a single template, and updates the PR template as well.

The new templates feature checklists, and wraps optional fields in a collapsible `<details>` tag.

## Screenshots / Architecture Diagrams <i>(Optional)</i>
### "Before" Screenshot
![image](https://user-images.githubusercontent.com/8496383/76976610-91819700-690a-11ea-87bd-d0a27dc77139.png)


### "After" Screenshot
![image](https://user-images.githubusercontent.com/8496383/76976552-80d12100-690a-11ea-95d8-289b24620763.png)


# Associated Issue(s)
fix #4

# How to test
1. Read the contents of the new issue templates:
  + Are there fields we're missing?
  + Are there some required things that should be optional?
  + Are there optional things that should be required?
  + Is there confusing language?
1. Copy the raw templates from this PR and create a new issue or submit a PR using them.  Did you encounter any ergonomic issues?

Because this is not executable code, no automated tests provided.

# Pull Request Submitter Checklist
+ [x] I have provided a description of what this PR changes
+ [x] I have linked an associated issue
+ [x] I have provided steps for how a tester can verify this PR works correctly
+ [x] I have included automated tests (or a description of why not)
+ [x] I have let issue stakeholders know that this PR is open and ready for their review

<details>
  <summary><b>Optional Submitter Items:</b></summary>

- [ ] I have added PR comments to interesting or potentially confusing parts of the code
- [ ] I have incremented the version number according to [semantic versioning](https://semver.org)
- [ ] I have met with issue stakeholders and personally walked them through changes
- [ ] I have created new GitHub issues for "TODO" lines added by this PR
- [ ] I have moved the associated issue to "in review" in Codetree

</details>

# Pull Request Reviewer Checklist
+ [ ] The PR Description sufficiently details changes
+ [ ] Issue(s) have been linked with this PR
+ [ ] Adequate testing steps are provided
+ [ ] There are automated tests included in the PR
+ [ ] I have pulled the source code and tested this PR locally
+ [ ] I have read through the code and provided one or more comments


<details>
  <summary><b>Optional Reviewer Items:</b></summary>

- [ ] If I am not the first reviewer, I have made my own checklist

</details>
